.github/workflows/release_community_cromwell.yml

```diff
@@ -10,6 +10,12 @@ jobs:
     steps:
     - uses: sbt/setup-sbt@v1
 
+    # Only Cromwell admins should be permitted to run this action
```

Comment:
I thought the bot running the GHA was a maintainer, not an admin of the repo, does that change anything?

Reply:
I think a human admin still kicks off the action as themselves, though.

Reply:
Yeah, as Adam said - a human is running the GHA (via workflow dispatch), part of what the GHA does is do git stuff as the Broadbot user. This check applies to the person kicking off the workflow. I didn't actually realize until this moment that we probably could have had all the git stuff done by the human rather than Broadbot... seems nice to use a bot identity for this type of automated process, though.